media-video/pipewire/pipewire-1.0.5-r1.ebuild

```diff
@@ -163,8 +163,7 @@ RDEPEND="
 DEPEND="${RDEPEND}"
 
 # TODO: Consider use cases where pipewire is not used for driving audio
```

Comment:
I think the TODO here can actually go entirely, redundant since we added USE=sound-server a while ago.